cmd/notificationsemail.go

```diff
@@ -154,26 +154,36 @@ var listProjectEmailsCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		if cmdProjectName == "" {
-			return fmt.Errorf("Missing arguments: project name is not defined")
+		if err := requiredInputCheck("Project name", cmdProjectName); err != nil {
+			return err
 		}
+
 		current := lagoonCLIConfig.Current
-		lc := client.New(
+		token := lagoonCLIConfig.Lagoons[current].Token
+		lc := lclient.New(
 			lagoonCLIConfig.Lagoons[current].GraphQL,
-			lagoonCLIConfig.Lagoons[current].Token,
-			lagoonCLIConfig.Lagoons[current].Version,
 			lagoonCLIVersion,
+			&token,
 			debug)
-		result, err := lagoon.GetProjectNotificationEmail(context.TODO(), cmdProjectName, lc)
+
+		result, err := l.GetProjectNotificationEmail(context.TODO(), cmdProjectName, lc)
 		if err != nil {
 			return err
 		}
+		if len(result.Name) == 0 {
+			outputOptions.Error = fmt.Sprintf("No project found for '%s'\n", cmdProjectName)
+		} else if len(result.Notifications.Email) == 0 {
```

Comment:
this will panic still, you'll need to check for `nil` before you can inspect the result of `Notifications`

Reply:
I'm only seeing it panic if the project doesn't exist, which we're checking prior to inspecting `Notifications` so we never hit it. Unless I've missed something

Reply:
does it return `[]`?

Reply:
Yep! And that gets caught on line 175

Reply:
Great!